src/js/port_handler.js

```diff
@@ -111,7 +111,8 @@ PortHandler.check_serial_devices = function () {
         } else {
             self.removePort();
             self.detectPort();
-            self.selectActivePort();
+            // already done in detectPort
+            // self.selectActivePort();
```

Comment:
That's not totally true... the detectPort() tries to detect any new device connected, the selectiveActivePort() selects one port in the list of availables. For this reason I think the port_handler.js needs a full refactor, it does things in a very complicated manner that now it's easier to do.
But I'm ok merging this, if something is broken, we can fix it later, when all the important PRs are merged and we refactor this.

Reply:
Yes, I can confirm that after selecting "I can't find my device" it does not change to the detected port, it maintains the "I can't find my device" option. I don't know if this comment is the culprit.

Reply:
same finding; had to manually select the newly found `Betaflight STM Electronics``

Reply:
Works on my computer :)


[Screencast from 20-05-24 16:08:55.webm](https://github.com/betaflight/betaflight-configurator/assets/8344830/2fd6b9de-b2b1-4866-bcb2-244f821cff7b)

What it does do is not call the function twice on startup:

![image](https://github.com/betaflight/betaflight-configurator/assets/8344830/0db4dcea-1315-4f99-9d2a-e791258da3bd)

Think this can be merged as we can revisit later